# Author: Ehsan Moravveji (VSC - KU Leuven)
# Purpose: The formatting of the 'license' token needs to be updated
#          The 'license-files' field does not belong to the 'project' section

diff -ruN dvc_studio_client-0.22.0-orig/pyproject.toml dvc_studio_client-0.22.0/pyproject.toml
--- dvc_studio_client-0.22.0-orig/pyproject.toml	2025-10-21 13:44:03.631039000 +0200
+++ dvc_studio_client-0.22.0/pyproject.toml	2025-10-21 13:45:05.002366000 +0200
@@ -6,8 +6,7 @@
 name = "dvc-studio-client"
 description = "Small library to post data from DVC/DVCLive to Iterative Studio "
 readme = "README.rst"
-license = "Apache-2.0"
-license-files = ["LICENSE"]
+license = { text = "Apache-2.0" }
 authors = [{name = "Iterative", email = "support@dvc.org"}]
 classifiers = [
   "Programming Language :: Python :: 3",
@@ -51,6 +50,9 @@
   "mypy==1.17.0"
 ]
 
+[tool.setuptools]
+license-files = ["LICENSE"]
+
 [tool.setuptools_scm]
 
 [tool.setuptools.packages.find]
